app/deploy/scripts/stop-service.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -x
+RES=`ps aux | grep -v grep | grep 'mint.jar' | awk '{print $2}'`
+if [[ ! -z "$RES" ]]; then
+    kill $RES
```

Comment:
TODO: this should be `docker stop` -- easier to do if `start-service.sh` uses the `--name` option to `docker run`